tip-wlan/values.yaml

```diff
@@ -1,31 +1,46 @@
-# Default values for Tip-Wlan
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+shared:
+  scalability: &jvm-options
```

Comment:
This block isn't used anywhere or do I miss something here?

Reply:
its not for now, but the idea is to have everything in values and override from resources\environments if\when necessary.